### ISSUE TITLE
Hue resets when selector is dragged to the right

### ISSUE THREAD

**Issue by stevenspasbo** [mswift42/theme-creator#5]
Chrome 42.0.2311.90 (64-bit) on Mac OSX 10.9.5

**Comment by stevenspasbo** [comment]
I said right, but meant left.

**Comment by mswift42** [comment]
Yes, this is a known issue with the angular-bootstrap-colorpicker. (See this [issue](https://github.com/buberdds/angular-bootstrap-colorpicker/issues/50))
